packages/front-end/components/Experiment/ChanceToWinColumn.tsx

```diff
@@ -34,10 +34,23 @@ export default function ChanceToWinColumn({
   baseline: SnapshotMetric;
   stats: SnapshotMetric;
 }) {
-  const minSampleSize = metric?.minSampleSize || defaultMinSampleSize;
-  const enoughData = hasEnoughData(baseline, stats, metric);
-  const suspiciousChange = isSuspiciousUplift(baseline, stats, metric);
-  const belowMinChange = isBelowMinChange(baseline, stats, metric);
+  const metricDefaults = useOrganizationMetricDefaults();
+
+  const minSampleSize =
+    metric?.minSampleSize || metricDefaults.minimumSampleSize;
+  const enoughData = hasEnoughData(baseline, stats, metric, metricDefaults);
+  const suspiciousChange = isSuspiciousUplift(
+    baseline,
+    stats,
+    metric,
+    metricDefaults
+  );
+  const belowMinChange = isBelowMinChange(
+    baseline,
+    stats,
+    metric,
+    metricDefaults
+  );
```

Comment:
These methods were changed to add the `metricDefaults` as an extra argument to be considered when performing the calculations rather than the hardcoded constants.